backend/src/main/kotlin/app/ehrenamtskarte/backend/cards/webservice/schema/CardMutationService.kt

```diff
@@ -370,7 +370,7 @@ class CardMutationService {
             context.request,
             dfe.field.name,
             cardHash,
-            activationResult.activationState != ActivationState.failed,
+            activationResult.activationState == ActivationState.success,
```

Comment:
Previously it was `!= ActivationState.failed` that means revoked was considered as success?